Templates/kafka-redis-nodejs.template/package.json

```diff
@@ -31,12 +33,16 @@
     "eslint-plugin-prettier": "^3.3.1",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^5.0.0",
-    "husky": "^5.0.9",
+    "husky": "4",
```

Comment:
why the downgrade?

Reply:
This 👇

"You're viewing documentation for husky v5, which is free to use in Open Source projects ❤️ and in early access for Sponsors 🎁. To use this new version at work, you can become a sponsor on GitHub Sponsors or Open Collective.

If you can't sponsor Husky, that's okay, husky v4 is free to use in any project."